Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM nginx:1.11.9-alpine
 
 # for htpasswd command
 RUN apk add --no-cache --update \
-      apache2-utils
+      apache2-utils \
+    && rm -f /etc/nginx/conf.d/*
```

Comment:
This `rm` command doesn't seems to be related to the above `apk` step. Could you separate it to another `RUN` step?

Reply:
Fixed.